### ISSUE TITLE
calcValue in hare beacon uses fnv

### ISSUE THREAD

**Issue by gavraz** [spacemeshos/go-spacemesh#1508]
hare/eligibility/beacon.go

The func calcValue (of the beacon) uses fnv instead of our agreed (low collision hash).
To my understanding, we still haven't agreed on such hash.
We use Hash32 for blocks and we also have Hash12. We should decide what we want to use for the beacon.

@barakshani any input on that?

**Comment by lrettig** [comment]
Will handle as part of work on self-healing

**Comment by lrettig** [comment]
This is partially addressed in #2394 but other usages still exist:

```
cmd/hare/hasher.go
4:      "hash/fnv"

hare/mock_oracle.go
8:      "hash/fnv"

hare/notifytracker.go
5:      "hash/fnv"

hare/haretypes.go
8:      "hash/fnv"

eligibility/fixedoracle.go
7:      "hash/fnv"
```
